walletclient.go

```diff
@@ -1,117 +1,151 @@
-// Package walletclient is a Go client for interacting with Spv Wallet.
 package walletclient
 
 import (
-	"github.com/bitcoin-sv/spv-wallet-go-client/transports"
+	"net/http"
+
+	"github.com/bitcoin-sv/spv-wallet/models"
 	"github.com/bitcoinschema/go-bitcoin/v2"
 	"github.com/libsv/go-bk/bec"
 	"github.com/libsv/go-bk/bip32"
 	"github.com/libsv/go-bk/wif"
 	"github.com/pkg/errors"
 )
 
-// ClientOps are used for client options
-type ClientOps func(c *WalletClient)
-
-// WalletClient is the spv wallet go client representation.
+// WalletClient is the spv wallet Go client representation.
 type WalletClient struct {
-	transports.TransportService
-	accessKey        *bec.PrivateKey
-	accessKeyString  string
-	transport        transports.TransportService
-	transportOptions []transports.ClientOps
-	xPriv            *bip32.ExtendedKey
-	xPrivString      string
-	xPub             *bip32.ExtendedKey
-	xPubString       string
+	signRequest     *bool
+	server          *string
```

Comment:
I'm not sure if storing pointers here makes sense. 
- It (wrongly) suggests that they are optional - but they are not. 
- If you really want to pass some struct field to a function, you can always do: `someFunc(&clientInstance.server)`

Reply:
dont think pointer suggest optional thing :) but if u really do not like it i can change it just pls define which way we go

Reply:
They do suggest that they are optional - at least there could be a situation when some of pointer is nil - and this forces us to check `if sth == nil` before usage.